### DISCUSSION TITLE
Two thing that should be in the documentation.

### DISCUSSION THREAD

**Discussion by antoniopaolini** [gsantner/markor#2116] [Ideas] [answered]
Hello, 
I'm looking for about filtered view in todo.txt but only diving into this "Discussion Section" I found the right info.

I think the info contained in #1940 but above all that contained in https://github.com/gsantner/markor/discussions/1646#discussioncomment-2409263 in particular the part that say: 

> For a different, filtered/sorted view, click on the filename/title on the top left.

should be in the docs, to be easily reachable.

I know, the #1940 is linked in the filtered view popup, but discovering the "tap on the title to filtered view" wasn't easy.

Not only this, but also the various long pressing/short pressing  function, etc.

Bye.

(sorry for my "heavy activity" here, I'm a new user and I really like Markor but i'm learning how it works).

**Comment by gsantner** [comment, marked as answer]
@antoniopaolini  
Unfortunatley there is nobody much active contributing to documentation. If you feel like something very important is missing, what you would add - please write the changes and make a merge request. Thank you! Btw, documentation is also just Markdown and contained in the same code repository here.
